appveyor.yml

```diff
@@ -0,0 +1,39 @@
+environment:
+
+  global:
+      PYTHON: "C:\\conda"
+      CMD_IN_ENV: "cmd /E:ON /V:ON /C .\\ci-helpers\\appveyor\\windows_sdk.cmd"
+      PYTHON_ARCH: "64" # needs to be set for CMD_IN_ENV to succeed. If a mix
+                        # of 32 bit and 64 bit builds are needed, move this
+                        # to the matrix section.
+
+  matrix:
+
+    - PYTHON_VERSION: "2.7"
+      CONDA_CHANNELS: "spyder-ide"
+      CONDA_DEPENDENCIES: "pytest qt5 pyqt5"
+    - PYTHON_VERSION: "2.7"
+      CONDA_DEPENDENCIES: "pytest qt pyqt"
+    - PYTHON_VERSION: "2.7"
+      CONDA_DEPENDENCIES: "pytest qt pyside"
+    - PYTHON_VERSION: "3.5"
+      CONDA_CHANNELS: "spyder-ide"
```

Comment:
This needs to be changed to `"msarahan"`:

https://anaconda.org/msarahan/qt5/files